packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/actions/access/AccessRestrictionUtil.java

```diff
@@ -0,0 +1,102 @@
+package org.alfresco.rest.actions.access;
```

Comment:
No idea if copyright notice here are needed, don't seem required and none of other tests have them (only 2 files, perhaps by accident)